app/views/container_topology/show.html.haml

```diff
@@ -7,7 +7,17 @@
         %input#box{'ng-change' => "show_hide_names(checkboxModel.value)",  'ng-model' => "checkboxModel.value", :type => "checkbox", 'ng-true-value' => 'true', 'ng-false-value' => 'false'}
         Display Names
       %i{'ng-click' => "refresh()", :class => "btn btn-default fa fa-refresh refresh-topology"}
-
+    .search-topology
+      %form.search-pf.has-button{:role => "form"}
+        .form-group.has-clear
+          .search-pf-input-group
+            %label.sr-only{:for => "search"} Search
+            %input#search.form-control{'placeholder' => "Search", 'type' => "search", "ng-model" => "query"}
```

Comment:
> Using ng-model with just a variable is a bit fragile, could you wrap it in an object please? ("search.query" instead of "query" for example) .. that's because if any DOM element between the ng-controller and this ng-model introduces a new scope, it will happily keep setting a different query variable than you can see on the controller's scope.

done